wordpress-mobile/WordPressAuthenticator-iOS
# Duplicate the login and signup flows

Closes #187 

This PR duplicates the classes found in the Login, Signup, and NUX folders. The old classes in those folders have been moved to a Legacy folder. All duplicated classes have the `Auth` prefix. 

### To test
1. In order to test the flow, download and apply [this patch](https://gist.github.com/mindgraffiti/c71a83f4d5662d48ee34afd110faa98a), using `git apply <path-to-patch>`. This will connect the duplicated flow to the Authenticator.
2. In WCiOS or WPiOS, change the podfile to use a local pod. The authenticator line will look similar to this: `pod 'WordPressAuthenticator', :path => '/Users/thuycopeland/WordPressAuthenticator-iOS/'`
3. Smoke-test flows and try to break it!

This PR does not change any class names (other than adding the prefix). It was easier to re-link classes using their existing names. Renaming will happen in the future.

Each ViewController in the duplicated flows should have a different colored background - seafoam green, pink, etc.
